tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is it needed? We are not using TypeScript in sortition pool tests. Unless we plan to do so eventually.

Reply:
Removed it in https://github.com/keep-network/sortition-pools/pull/127/commits/4ba45853abf00e3486568229a9bbddac8475f0bb.